docs/examples/call-create-room-json-api.swadl.yaml

```diff
@@ -0,0 +1,18 @@
+id: create-room-request-workflow
+activities:
```

Comment:
isn't the example with the exchange rate api enough in the end?

Reply:
exchange rate api is a GET and does not show usage of the request body. But we can keep this one.